### DISCUSSION TITLE
It would be good to have the accessibility option that allows the form that is activated with alt+space to be less blurry and its space to be smaller

### DISCUSSION THREAD

**Discussion by aurelio20000** [Flow-Launcher/Flow.Launcher#2780] [Polls]
It would be good to have the accessibility option that allows the form that is activated with alt+space to be less blurry and its space to be smaller

**Comment by deefrawley** [comment]
Hi, have you explored the Appearance settings in Flow? You can change the window size there as well as play with themes.

**Reply by aurelio20000** [reply]
Yes, I have done it, but it would be nice if it was a more minimalist presentation that doesn't take up too much desktop space.

**Reply by z1nc0r3** [reply]
Hi @aurelio20000, what are your suggestions for a minimalist presentation? Do you mean small font sizes, decreased width, height, etc.?
